core/src/components/alert/alert.scss

```diff
@@ -184,7 +186,7 @@
   text-align: start;
   appearance: none;
 
-  contain: strict;
+  contain: content;
```

Comment:
Without this, each radio/checkbox will still be sized at 48px. This change allows the height of the element to increase as the text wraps.